### PR TITLE
tests: fix test_down_peers.

### PR DESCRIPTION
**Pull request by huachaohuang**

### CODE REVIEW

File: tests/raftstore/test_down_peers.rs [hunk 1]
```diff
@@ -31,17 +31,36 @@ fn test_leader_down_and_become_leader_again<T: Simulator>(cluster: &mut Cluster<
     let node = cluster.leader_of_region(1).unwrap();
     let node_id = node.get_id();
     cluster.add_send_filter(Isolate::new(node_id));
+    debug!("node: {:?}", node);
 
     // Kill another node.
-    let next_id = if node_id < count { node_id + 1 } else { 1 };
+    let next_id = if node_id < count {
```
Comment (siddontang) [inline tests/raftstore/test_down_peers.rs]: do we need to update rustfmt?

**Comment by siddontang** [comment]
Rest LGTM

**Comment by siddontang** [comment]
PTAL @overvenus

**Comment by overvenus** [comment]
LGTM

**Pull request merged**